en/flight_controller/omnibus_f4_sd.md

```diff
@@ -171,12 +171,19 @@ Instructions for this are provided in the [TBS Crossfire Manual](https://www.tea
 
 The schematics are provided by [Airbot](https://myairbot.com/): [OmnibusF4-Pro-Sch.pdf](http://bit.ly/obf4pro).
 
-## Build & Upload Instructions
+## Build & Upload Instructions{#upload}
 
 The board comes pre-installed with [Betaflight](https://github.com/betaflight/betaflight/wiki). 
-Before PX4 firmware can be installed, the PX4 bootloader needs to be built and uploaded.
+Before the PX4 firmware can be installed, the PX4 bootloader needs to be flashed.
 
-### Download
+There are two options for flashing the bootloader, via Betaflight Configuator (easier), or building from source (guaranteed to work).
+
+### Via Betaflight Configurator
+First download the pre-built bootloader binary [omnibusf4sd_bl.hex](https://github.com/PX4/px4_user_guide/raw/master/assets/flight_controller/omnibus_f4_sd/omnibusf4sd_bl_d52b70cb39.hex).
+Then flash it via Betaflight Configurator.
```

Comment:
This is probably not enough. We can link to one of:
- https://github.com/betaflight/betaflight/wiki/Installing-Betaflight
- https://oscarliang.com/flash-update-betaflight/
And mention to use 'Load local firmware.'
What do you think @hamishwillee ?

Reply:
I think both of those cover too much unrelated material. I've included a simple install instruction in the flight controller docs, with screenshot. Please check - I can't actually run it without a board.